project/gh-action/infrastructure/kubernetes/deployment.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: web-application
+  
+spec:
+  replicas: "3"  
+  selector:
+    matchLabels:
+      app: frontend 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)